modules/components/dropdown/tag-input-dropdown.scss

```diff
@@ -0,0 +1,21 @@
+/deep/ .ng2-dropdown-menu {
+  width: 388px !important;
+  padding: 0 !important;
+  border: 1px solid #359cbf;
+}
+
+.dropdown-bottom-button {
+  height: 30px;
+  border-top: 1px solid #359cbf;
+
+  display: flex;
+  align-items: center;
+}
+
+.dropdown-bottom-text {
+  font-size: 14px;
+  font-weight: bold;
+  color: #359cbf;
+  padding-left: 15px;
+  cursor: pointer;
```

Comment:
Add the `cursor: pointer` to `.dropdown-bottom-button` instead because the click event is on the whole div